apiserver/backrestservice/backrestimpl.go

```diff
@@ -469,6 +477,14 @@ func Restore(request *msgs.RestoreRequest, ns, pgouser string) msgs.RestoreRespo
 		return resp
 	}
 
+	//check that the current cluster is not currently upgraded to the deployed
+	//Operator version. If not, do not allow the command to completes
```

Comment:
Typo: `completes` to `complete`

Reply:
Noticed a bunch of other instances of this one as well.

Reply:
Yep, reworked that comment to read:
```
//check if the current cluster is not upgraded to the deployed
//Operator version. If not, do not allow the command to complete
```
and fixed in all instances. Thanks!